sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectKey=twilio_twilio-cli
+sonar.projectName=twilio-cli
+sonar.organization=twilio
+
+sonar.sources=src/
+sonar.tests=test/
+# sonar.test.exclusions= < No exclusions currently >
```

Comment:
nit: If not required, we can remove this.

Reply:
I've deliberately added this to avoid checking documentation back again for skipping some specific test branches.